packages/editor/.eslintrc.js

```diff
@@ -0,0 +1 @@
+module.exports = require('../../config/eslintrc.base');
```

Comment:
Just leaving a comment here so that we don’t forget to update this the same way we did in https://github.com/mongodb-js/mongosh/commit/c4372550cc02878738319f7767e9ff896491fed4 :)